runtimes/runtimes/standalone.ts

```diff
@@ -102,6 +106,26 @@ export const standalone = (props: RuntimeProps) => {
         } else {
             lspConnection.console.info('Runtime: Initializing runtime without encryption')
             auth = new Auth(lspConnection)
+
+            chat = {
```

Comment:
Is moving initialisation of chat object out of `initializeRuntime` necessary? It still can be done inside in the condition